### PR TITLE
Update alluxio version to 2.9.0

### LINKED ISSUE #7811: Update alluxio version to 2.9.0
**Describe the bug**
We are updating our docker image to default to alluxio 2.9.0, we should update the default in our code.  Ideally this isn't set via env variable, but rather a config.

### PR DESCRIPTION
**Pull request by res-life**
Closes #7811 
Signed-off-by: Chong Gao <res_life@163.com>

Set default Alluxio home as /opt/alluxio
The docker file should add the following before start Alluxio
```
RUN ln -s /opt/alluxio-x.y.z /opt/alluxio
```
Refer to: https://github.com/NVIDIA/spark-rapids-container/pull/39

In the future do not need to update the default Alluxio home path in the code.

### CODE REVIEW

**Comment by res-life** [comment]
Please review the solution first.

File: docs/configs.md [hunk 1]
```diff
@@ -31,8 +31,9 @@ scala> spark.conf.set("spark.rapids.sql.concurrentGpuTasks", 2)
...
 <a name="alluxio.bucket.regex"></a>spark.rapids.alluxio.bucket.regex|A regex to decide which bucket should be auto-mounted to Alluxio. E.g. when setting as "^s3://bucket.*", the bucket which starts with "s3://bucket" will be mounted to Alluxio and the path "s3://bucket-foo/a.csv" will be replaced to "alluxio://0.1.2.3:19998/bucket-foo/a.csv". It's only valid when setting spark.rapids.alluxio.automount.enabled=true. The default value matches all the buckets in "s3://" or "s3a://" scheme.|^s3a{0,1}://.*|Runtime
+<a name="alluxio.home"></a>spark.rapids.alluxio.home|The Alluxio installation home path or link to the installation home path. You can create a link to Alluxio installation path, e.g: ln -s /opt/alluxio-x.y.z /opt/alluxio|/opt/alluxio|Startup
```
Comment (tgravescs) [inline docs/configs.md:36]: I don't think we need the last sentence about creating a symlink.  User can figure that out on their own or via our instructions

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala [hunk 2]
```diff
@@ -22,22 +22,17 @@ import scala.io.BufferedSource
 
 /**
  * Alluxio master address and port reader.
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala:24]: needs copyright updated

**Comment by tgravescs** [comment]
Has this been tested end to end with docker file?  If not please test on databricks

File: docs/configs.md (hunk 1 above)
Comment (res-life) [reply docs/configs.md]: done

File: sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala (hunk 2 above)
Comment (res-life) [reply sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala:24]: done

**Comment by res-life** [comment]
Tested successfully.

**Comment by res-life** [comment]
build

**Pull request merged**